exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/work_in_progress/text_ai/txaie_init.ipynb

```diff
@@ -26,7 +26,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "%run ../utils/access_store_ui.ipynb\n",
+    "%run ../../utils/access_store_ui.ipynb\n",
```

Comment:
Ah - UI is not covered by NB tests, right?

Reply:
yes